### ISSUE TITLE
Number of parameters of WX_WideResNet_STL model

### ISSUE THREAD

**Issue by tqtrunghnvn** [maple-research-lab/EnAET#4]
For STL-10, ```WX_WideResNet_STL``` model only contains 5.93 million parameters while this number in the MixMatch paper is 23.8 million parameters.
Could you clarify this point?

**Comment by wang3702** [comment]
Thanks for your inquiry! I don't know why this happens. But I adopted the architecture from https://github.com/xternalz/WideResNet-pytorch/blob/master/wideresnet.py. It's also possible that the difference raised by the implementation of tensorflow and pytorch.

**Comment by tqtrunghnvn** [comment]
Thank you for your quick reply!
Did you use this model (```WX_WideResNet_STL``` with 5.93 million parameters) to achieve 8.04 Top-1 accuracy on STL-10 (1000 labeled samples)?

**Comment by wang3702** [comment]
Yes. I believe you can run and achieve similar accuracy, may be slightly different based on your environment.

**Comment by tqtrunghnvn** [comment]
I see. Thank you so much for your contribution!
